numpy/distutils/mingw32ccompiler.py

```diff
@@ -53,7 +53,9 @@ def get_msvcr_replacement():
 _TABLE = re.compile(r'^\s+\[([\s*[0-9]*)\] ([a-zA-Z0-9_]*)')
 
 # the same as cygwin plus some additional parameters
-class Mingw32CCompiler(distutils.cygwinccompiler.CygwinCCompiler):
+_parent = distutils.cygwinccompiler.CygwinCCompiler # super() useable for all
+                                                    # python versions?
```

Comment:
`super(Mingw32Compiler, self)` works on all python versions we care about, so better to do it that way